File: DYNAMO.py
for t in range(int(input().rstrip())):
    N = int(input().rstrip())
    temp = 10**N
    S = 2*temp
    A = int(input().rstrip())
    S += A
    print(S, flush=True)
    B = int(input().rstrip())
    C = temp-B
    print(C, flush=True)
    D = int(input().rstrip())
    E = temp-D
    print(E, flush=True)
    flag = int(input().rstrip())
    if flag == -1:
        break

File: DYNAMO.cpp
#include <bits/stdc++.h>
#define MOD 1000000007
#define MP make_pair
#define PB push_back
#define EB emplace_back
#define FOR(i,a,b) for (long long i = a; i < b; i++)
#define REV(i,a,b) for (long long i = a; i > b; i--)
typedef std::vector<long long> vi;
typedef long long ll;
typedef unsigned long long ull;
using namespace std;

int main() {
    int t;
    cin >> t;
    while(t--) {
        ll N, S, A, B, C, D, E;
        cin >> N;
        ll temp = ceil(pow(10, N));
        S = 2 * temp;
        cin >> A;
        S += A;
        cout << S << "\n" <<flush;
        cin >> B;
        C = temp-B;
        cout << C << "\n" << flush;
        cin >> D;
        E = temp-D;
        cout << E << "\n" << flush;
        int flag;
        cin >> flag;
        if(flag==-1) {
            break;
        }
    }
    return 0;
}
